extra/modules/greenbids-real-time-data/src/test/java/org/prebid/server/hooks/modules/greenbids/real/time/data/v1/FilterServiceTest.java

```diff
@@ -0,0 +1,137 @@
+package org.prebid.server.hooks.modules.greenbids.real.time.data.v1;
+
+import ai.onnxruntime.OnnxTensor;
+import ai.onnxruntime.OnnxValue;
+import ai.onnxruntime.OrtException;
+import ai.onnxruntime.OrtSession;
+import ai.onnxruntime.TensorInfo;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.prebid.server.exception.PreBidException;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.data.ThrottlingMessage;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor.FilterService;
+import org.prebid.server.hooks.modules.greenbids.real.time.data.model.predictor.OnnxModelRunner;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.AbstractMap;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class FilterServiceTest {
+
+    @Mock
+    private OnnxModelRunner onnxModelRunnerMock;
+
+    @Mock
+    private OrtSession.Result results;
+
+    @Mock
+    private OnnxTensor onnxTensor;
+
+    @Mock
+    private TensorInfo tensorInfo;
+
+    private FilterService target;
+
+    @BeforeEach
+    public void setUp() {
+        target = new FilterService();
+    }
+
+    @Test
+    public void shouldReturnFilteredBiddersWhenValidThrottlingMessagesProvided() throws OrtException, IOException {
```

Comment:
the test case name pattern is `<method_under_testing_name>Should<expected_behavior>When<condition>`

Reply:
fixed names according to pattern